project/GenTypeClass.scala

```diff
@@ -93,8 +93,10 @@ object TypeClass {
   lazy val monadTell = TypeClass("MonadTell", |*->*|->*, extendsList = Seq(monad))
   lazy val monadReader = TypeClass("MonadReader", |*->*|->*, extendsList = Seq(monad), createSyntax = false)
   lazy val comonadStore = TypeClass("ComonadStore", |*->*|->*, extendsList = Seq(comonad), createSyntax = false)
-
+  
   lazy val bindRec = TypeClass("BindRec", *->*, extendsList = Seq(bind))
+  lazy val cofoldable = TypeClass("Cofoldable", |*->*|->*, extendsList = Seq(cofoldable1), createSyntax = true)
```

Comment:
Why not `*->*`?

Reply:
The type is `trait Cofoldable[F[_], A] extends Cofoldable1[F, A]` seems the same as say, MonadTell, which also has `|*->*|->*`

Reply:
Agreed, this seems wrong to me. It should just be `F[_]`.

Reply:
There could be a case for `Cofoldable[F, A]` (so that is extends `Reducer[F, A]`) and then we could have instances like `Cofoldable[String, Char]`. Also would avoid duplication between the two type classes.